SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -196,46 +186,47 @@ if ($PsBoundParameters.ContainsKey("shutdown") -or $PsBoundParameters.ContainsKe
                                     if (Test-vSphereAuthentication -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass) {
-                                        $listofVmsnotvCLS = (Get-VM -Location $clustername_extra | Select-Object Name, PowerState, @{N = "ToolsStatus"; E = { $_.ExtensionData.Guest.ToolsStatus } } | Where-Object { $_.name -inotmatch "vCLS" -and $_.PowerState -eq "PoweredOn" })
-                                        $vmnames = $listofVmsnotvCLS.Name
-                                        $tools = $listofVmsnotvCLS.ToolsStatus
-                                        foreach ($vmname in $vmnames) {
-                                            if ($tools[$vmnames.IndexOf($vmname)] -eq "toolsOK") {
-                                                Stop-CloudComponent -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass -nodes $vmname -timeout 300
+                                        # TODO Fix variable name and matching condition
+                                        $listOfVMsNotVcls = (Get-VM -Location $clusterNameExtra | Select-Object Name, PowerState, @{N = "ToolsStatus"; E = { $_.ExtensionData.Guest.ToolsStatus } } | Where-Object { $_.name -inotmatch "vCLS" -and $_.PowerState -eq "PoweredOn" })
+                                        $vmNames = $listOfVMsNotVcls.Name
+                                        $tools = $listOfVMsNotVcls.ToolsStatus
+                                        foreach ($vmName in $vmNames) {
+                                            if ($tools[$vmNames.IndexOf($vmName)] -eq "toolsOK") {
+                                                Stop-CloudComponent -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass -nodes $vmName -timeout 300
                                             } else {
-                                                Write-Error "Unable to shutdown virtual machines $vmname. VMware Tools is not running, Please shutdown the virtual machines before retrying. Exiting..."
+                                                Write-Error "Unable to shutdown virtual machines $vmName. VMware Tools is not running, Please shutdown the virtual machines before retrying. Exiting..."
                                                 Exit
                                             }
                                         }
 
                                         # Shut Down the vSphere Cluster Services Virtual Machines
-                                        Set-Retreatmode -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass -cluster $clustername -mode enable
+                                        Set-Retreatmode -server $vcfVcenterDetails.fqdn -user $vcfVcenterDetails.ssoAdmin -pass $vcfVcenterDetails.ssoAdminPass -cluster $clusterName -mode enable
```

Comment:
Shuld we use `Set-RetreatMode` in the module for the function name?

Reply:
There are several functions that needs to be changed. I'll do those in a separate MR

Reply:
There are several functions that needs proper camelCase. Noted and will be fixed in another MR